packages/web/src/scss/helpers/links/_links.scss

```diff
@@ -1,3 +1,11 @@
+// 1. Reset button styles for individual link variants.
+// 2. Generate link colors for all link states.
+// 3. Links in headings always have the default decoration.
+// 4. Allow link underline everywhere, except for headings in default state.
+// 5. Disable pointer and text decoration changes for disabled links.
+// 6. Allow link stretching over the closest element with `position: relative`.
+// 7. Only links in headings have visited state.
```

Comment:
Sorry to bother you with this question but why only links in headings have visited state?

Reply:
That's how it's designed 🤷🏻.

Reply:
That's a design decision. Actually from Jobs Designers :) They don't want to have visited styles anywhere except in headings, because it is used to let the user know they already saw the job detail.

Reply:
Aah. 🙈

I would suppose this could be a link variant. I can image use case where I would like to reflect visited state of the link (eg. TOC, dictionary, …).

Reply:
Thank you @crishpeen for a more polite explanation 🙂.

Reply:
I improved the comment.